packages/react-components/react-text/README.md

```diff
@@ -27,27 +21,9 @@ const App = () => (
 );
 ```
 
-Or by installing only the `@fluentui/react-text` package. Keep in mind you'll need to install the FluentProvider package as well:
-
-```sh
-npm install @fluentui/react-text
-npm install @fluentui/react-provider
-```
-
-```jsx
-import { FluentProvider } from '@fluentui/react-provider';
-import { Text } from '@fluentui/react-text';
-
-const App = () => (
-  <FluentProvider>
-    <Text>Fluent UI Text!</Text>
-  </FluentProvider>
-);
-```
-
 ## Typography wrappers
 
-![List of typography variants by sorted descending by size](./assets/typography-examples.gif 'Typography wrapper list')
+![List of typography variants by sorted descending by size](./assets/typography-examples.png 'Typography wrapper list')
```

Comment:
Updated image to contain the new typography wrapper `Caption2` introduced in https://github.com/microsoft/fluentui/pull/22934